### PR TITLE
Update changelog for 20.1.0

### PR DESCRIPTION
**Pull request by kritika-singh3**
Also refactored materials group so that the webhook APIs have better visibility

### CODE REVIEW

**Comment by kritika-singh3** [comment]
Preview:
Screen Shot 2020-01-30 at 11 26 31 AM

Screen Shot 2020-01-30 at 11 26 36 AM

File: source/includes/_050-materials-webhook.md.erb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Notify Materials
```
Comment (bdpiparva) [inline source/includes/_050-materials-webhook.md.erb]: Can we say webhook in title?

Comment (kritika-singh3) [reply source/includes/_050-materials-webhook.md.erb]: Not sure. We haven't used it in any doc.

Comment (bdpiparva) [reply source/includes/_050-materials-webhook.md.erb]: Webhook and notify are two different APIs

### Notify API
https://github.com/gocd/gocd/blob/5559396a8164083734c05b0d211d461496d73d61/spark/spark-base/src/main/java/com/thoughtworks/go/spark/Routes.java#L185-L191

### Webhook API
https://github.com/gocd/gocd/blob/5559396a8164083734c05b0d211d461496d73d61/spark/spark-base/src/main/java/com/thoughtworks/go/spark/Routes.java#L867-L873

Comment (kritika-singh3) [reply source/includes/_050-materials-webhook.md.erb]: So the webhook API you are mentioning isn't documented, isn't it

Comment (bdpiparva) [reply source/includes/_050-materials-webhook.md.erb]: It is documented [here](https://api.gocd.org/20.1.0/#hosting-your-repository-on-bitbucket)

Screenshot 2020-01-30 at 12 07 32 PM

Comment (kritika-singh3) [reply source/includes/_050-materials-webhook.md.erb]: Sorry missed it.

Comment (kritika-singh3) [reply source/includes/_050-materials-webhook.md.erb]: Added another header `Webhook` which contains the above mentioned APIs.

**Pull request merged**